src/tools/test-targets.ts

```diff
@@ -39,3 +42,154 @@ export const listTestTargets = async (options: ListOptions): Promise<void> => {
     );
   }
 };
+
+export type TestTargetPullData = components["schemas"]["TestTargetPullData"];
+
+type TestCases = components["schemas"]["TestTargetPullData"]["testCases"];
+
+const writeYaml = (data: TestTargetPullData, destination?: string) => {
+
+  for(const testCase of data.testCases) {
+    const folderName = buildFolderName(testCase, data.testCases, destination);
+    const testCaseFilename = buildFilename(testCase, folderName);
+    // Ensure folder exists
+    fs.mkdirSync(folderName, { recursive: true });
+    fs.writeFileSync(path.join(folderName, testCaseFilename), yaml.stringify(testCase));
+  }
+
+  //fs.writeFileSync("test-target.yaml", yaml.stringify(data));
+};
+
+export const buildFolderName = (testCase: TestCases[number], testCases: TestCases, destination?: string): string => {
+  type T = { id?: string | number; prerequisiteId?: string | number; description?: string };
+
+  const toSegment = (desc: string | undefined): string => {
+    const raw = String(desc ?? "root");
+    const normalized = raw
+      .normalize("NFKD")
+      .replace(/[\u0300-\u036f]/g, "");
```

Comment:
are these illegal characters in file names? (maybe if yes we can add a constant with a speaking name ;))